app/src/main/java/com/example/lastfm/LastFMClient.java

```diff
@@ -12,6 +12,8 @@
 
     @GET("?method=chart.gettopartists")
     Single<List<Artist>> getArtists(@Query("limit")int numberArtists, @Query("api_key")String key, @Query("format")String format);
+    @GET("?method=artist.getinfo")
+    Single<List> getArtistInfo(@Query("artist")String name,@Query("api_key")String key,@Query("format")String format);
```

Comment:
why do u get only the list without any type?

Reply:
Changed List to ArtistInfo.